Multisig/UI/Transactions/TransactionsList/TransactionsViewModel.swift

```diff
@@ -89,7 +89,6 @@ class TransactionsViewModel: BasicLoadableViewModel {
             .receive(on: RunLoop.main)
             .sink(receiveCompletion: { completion in
                 if case .failure(let error) = completion {
-                    self.nextURL = nil
```

Comment:
But this way if you scroll again to bottom it will trigger the pagination again, is it correct to do so ?

Reply:
I would assume this behavior. Otherwise, you can't re-trigger pagination at all, apart from refreshing the whole view.

Reply:
if it could decide based on the response's nextURL, and not on failure, then it would be ideal

Reply:
what do you mean?

Reply:
nevermind, it is already good (on the line 79)